cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -52,6 +58,17 @@ def _init_options(self, kwargs):
         if "options" not in self.options:
             self.options["options"] = {}
 
+        # processes needs to be an integer.
+        if "processes" in self.options:
+            try:
+                self.options["processes"] = int(self.options["processes"])
+            except ValueError:
+                raise TaskOptionsError(
+                    f"'processes' value '{self.options['processes']}' is not an integer"
+                )
+        else:
+            self.options["processes"] = 1
```

Comment:
```suggestion
        try:
            self.options["processes"] = int(self.options.get("processes", 1))
        except (TypeError, ValueError):
            raise TaskOptionsError("Please specify an integer for the `processes` option.")
```

Reply:
Yeah, that makes sense.  Did you purposefully change the wording on the exception that is raised?

Reply:
It's a suggestion; take it or leave it -- generally I prefer to word things so we're giving the user a clue about what they need to change rather than just telling them what they did wrong.